docs/source/developer_guide/guides/1_simple_python_stage.md

```diff
@@ -19,7 +19,7 @@ limitations under the License.
 
 ## Background
 
-Morpheus makes use of the MRC graph-execution framework. Morpheus pipelines are built on top of MRC pipelines. Pipelines in MRC are made up of segments; however, in many common cases, a MRC pipeline will consist of only a single segment. Our Morpheus stages will interact with the MRC segment to build nodes and add them to the MRC graph. In the common case, a Morpheus stage will add a single node to the graph, but in some cases it will add multiple nodes to the graph.
+Morpheus makes use of the MRC graph-execution framework. Morpheus pipelines are built on top of MRC pipelines, which are comprised of collections of nodes and edges, called segments (think sub-graphs),  which can in turn be connected by ingress/egress ports. In many common cases, an MRC pipeline will consist of only a single segment. Our Morpheus stages interact with the MRC segment to define, build, and add nodes to the MRC graph; the stages themselves can be thought of as packaged units of work to be applied to data flowing through the pipeline. These work units comprising an individual Morpheus stages may consist of a single MRC node, a small collection of nodes, or an entire MRC subgraph.
```

Comment:
The last sentence reads "Morpheus stages" (plural) when it should probably read "Morpheus stage" (singular).